CMakeLists.txt

```diff
@@ -76,6 +76,14 @@ if(HAS_LIBDISPATCH_API)
   find_package(Threads REQUIRED)
 endif()
 
+# CMake's Threads adds '-pthread' flag to the interface link
+# libraries, which isn't supported by Swift. This is not enabled
+# when building with MSVC, but it trips up the Android build, so
+# we need to clear out the threads INTERFACE_LINK_LIBRARIES.
+if (CMAKE_SYSTEM_NAME STREQUAL "Android")
+  set_property(TARGET Threads::Threads PROPERTY INTERFACE_LINK_LIBRARIES "")
```

Comment:
I think this probably addresses [the same issue I fixed in `build-script` years ago](https://github.com/apple/swift/commit/a4ff4db508690711ecf62d2d8aa5bb5ece80d8bf#diff-1a5a064af373f742294e3b312a5c8d0ab52f5aef78052490e6005165472cda48R2424). Maybe it would be better to add that CMake flag to `build.ps1`?

Reply:
I resolved it differently now, thanks to @compnerd 's suggestion.